entrypoint.sh

```diff
@@ -1,8 +1,25 @@
 #!/bin/sh
+version() {
+  if [ -n "$1" ]; then
+    echo "-v $1"
+  fi
+}
+
+install_extensions() {
+  if [ -n "$1" ]; then
+    echo "$1" | sed "s/[^ ][^ ]*/'&'/g" | xargs gem install
+  else
+    gem install -N rubocop-rails rubocop-performance rubocop-rspec rubocop-i18n rubocop-rake
+  fi
+}
 
 cd "$GITHUB_WORKSPACE"
 
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
+gem install rubocop $(version $INPUT_RUBOCOP_VERSION)
```

Comment:
```suggestion
gem install -N rubocop $(version $INPUT_RUBOCOP_VERSION)
```

Reply:
We cannot cpecify gem version with -N flag

Reply:
The `-N` argument is used to disable documentation generation.
https://guides.rubygems.org/command-reference/#gem-install

Reply:
Sorry here, I misunderstood everything and everything works with the flag -N